Mathlib/MeasureTheory/Measure/FiniteMeasure.lean

```diff
@@ -645,7 +645,7 @@ theorem tendsto_iff_forall_integral_tendsto {γ : Type*} {F : Filter γ} {μs :
   simp_rw [aux, BoundedContinuousFunction.toReal_lintegral_coe_eq_integral] at tends_pos tends_neg
   exact Tendsto.sub tends_pos tends_neg
 
-lemma continuous_integral_boundedContinousFunction
+lemma continuous_integral_boundedContinuousFunction
```

Comment:
You added deprecated aliases for other renamed lemmas but not for this one. Is it intentional?
bors d+

Reply:
This one xas added a few day ago by myself.< It's ony usd in PFR

Reply:
Yes, it's so recent that it's not worth deprecating.